server/routes/createPlan/personalInterests/personalInterestsController.test.ts

```diff
@@ -177,16 +184,31 @@ describe('PersonalInterestsController', () => {
       expect(res.redirect).toHaveBeenCalledWith(addressLookup.createPlan.checkYourAnswers(id))
     })
 
-    it('On success - mode = update - calls api and redirects to learning profile', async () => {
+    it('On success - mode = update - calls CIAG api and redirects to learning profile', async () => {
+      config.featureToggles.useNewInductionApiEnabled = false
       req.context.plan = { skillsAndInterests: {} }
-      req.body.personalInterests = PersonalInterestsValue.OTHER
+      req.body.personalInterests = [PersonalInterestsValue.OTHER]
```

Comment:
Ah, looks like you found the same problem in the existing tests as I did with Skills (in that the request body field should be an array rather than simple string) 👍